web/blog/tenzir-v4.2/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Tenzir v4.2
+authors: [Dakostu, mavam]
+date: 2023-09-07
+tags: [release, pipelines, connectors, s3, zmq]
+draft: true
+---
+
+We've just released Tenzir v4.2 that introduces two new connectors: S3 for
+interacting with blob storage and [ZeroMQ][zeromq] for writing distributed
+multi-hop pipelines.
+
+[zeromq]: https://zeromq.org/
+
+<!--truncate-->
+
+## S3 Saver & Loader
+
+TODO
```

Comment:
I should probably do this. 
After this PR has been merged - then I add my S3 stuff to this file in my PR? @mavam

Reply:
Exactly what I was thinking.